projects/canopy/src/lib/card/card.component.ts

```diff
@@ -32,19 +39,36 @@ let nextUniqueId = 0;
 export class LgCardComponent implements AfterContentInit, OnDestroy {
   private subscription: Subscription;
   private uniqueId = nextUniqueId++;
+  orientationClasses: Array<string> = [];
 
   @ContentChild(forwardRef(() => LgButtonToggleDirective))
   buttonToggle: LgButtonToggleDirective;
   @ContentChild(forwardRef(() => LgCardToggableContentComponent))
   cardToggableContent: LgCardToggableContentComponent;
   @ContentChild(forwardRef(() => LgCardNavigationTitleComponent))
   cardNavigationTitle: LgCardNavigationTitleComponent;
+  @Input() variant: CardVariant = 'default';
 
   constructor(private renderer: Renderer2, private hostElement: ElementRef) {}
 
+  @Input()
+  set orientation(orientation: OrientationResponsive) {
+    this.orientationClasses = this.toggleClasses(
+      this.createNewOrientationClasses(orientation),
+      this.orientationClasses,
+    );
+  }
+  @HostBinding('class.lg-card--orientation') get orientationClass(): boolean {
+    return this.orientationClasses.length !== 0;
```

Comment:
Could this be only `return !this.orientationClasses.length`?

Reply:
yep, Done :)